src/Admin/AbstractAdmin.php

```diff
@@ -3261,6 +3251,14 @@ public function canAccessObject($action, $object)
         return $this->hasAccess($action, $object);
     }
 
+    /**
+     * @return MutableTemplateRegistryInterface
+     */
+    final public function getTemplateRegistry()
+    {
+        return $this->templateRegistry;
```

Comment:
This can be null

Reply:
This object are create in DI. It can be null only when people don't register `Admin`

Reply:
It can also be null if I do "new Admin()"

Reply:
Add `hasTemplateRegistry()` method and throw LogicException in `getTemplateRegistry()` should resolve this problem.